lib/realtime/tenants/replication_connection.ex

```diff
@@ -339,7 +339,7 @@ defmodule Realtime.Tenants.ReplicationConnection do
 
   @impl true
   def handle_disconnect(state) do
-    Logger.warning("Disconnecting broadcast changes handler: #{inspect(state, pretty: true)}")
+    Logger.warning("Disconnecting broadcast changes handler in the step : #{state.step}")
```

Comment:
maybe inspect is missed

Reply:
it's an atom but yeah... better do it just in case 😅